### ISSUE TITLE
kuberspray run failed

### ISSUE THREAD

**Issue by reactima** [MusicDin/kubitect#21]
vm-master and vm-workers machines were created ubuntu 20.04. 
kubitect.yaml was copy&pasted from https://kubitect.io/user-guide/getting-started/#step-44-kubernetes-properties

however kuberspray run failed as the following:  

TASK [cluster-config/import : Include cluster config as variable] **************
ok: [127.0.0.1]

TASK [kubespray-clone : Clone Kubespray git project] ***************************
fatal: [127.0.0.1]: FAILED! =>
  msg: |-
    The task includes an option with an undefined variable. The error was: 'dict object' has no attribute 'url'

    The error appears to be in '/root/.kubitect/clusters/my-first-cluster/ansible/kubitect/roles/kubespray-clone/tasks/main.yaml': line 3, column 3, but may
    be elsewhere in the file depending on the exact syntax problem.

    The offending line appears to be:

    ---
    - name: Clone Kubespray git project
      ^ here

NO MORE HOSTS LEFT *************************************************************

PLAY RECAP *********************************************************************
127.0.0.1                  : ok=1    changed=0    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0

Error while running ansible-playbook: Error during command execution: ansible-playbook error: one or more host failed

Command executed:  /root/.kubitect/clusters/my-first-cluster/bin/venvs/main/bin/ansible-playbook --extra-vars {"kubitect_cluster_path":"/root/.kubitect/clusters/my-first-cluster"} --inventory 127.0.0.1, --connection local /root/.kubitect/clusters/my-first-cluster/ansible/kubitect/kubespray-setup.yaml

**Comment by reactima** [comment]
content of /root/.kubitect/clusters/my-first-cluster/ansible/kubitect/roles/kubespray-clone/tasks/main.yaml

---
- name: Clone Kubespray git project
  git:
    repo: "{{ config.kubernetes.kubespray.url }}"
    version: "{{ config.kubernetes.kubespray.version }}"
    dest: "{{ kubitect_cluster_path }}/ansible/kubespray"
    single_branch: yes
    depth: 1
    force: yes#

**Comment by MusicDin** [comment]
Hi,

thanks for reporting this issue. It seems that default Kubespray URL is not set.
I will fix this as soon as possible. In the meantime, explicitly specifying the Kubespray URL should fix the problem:
```yaml
...
kubernetes:
  version: "v1.22.6"
  networkPlugin: "calico"
  dnsMode: "coredns"
  kubespray:
    url: "https://github.com/kubernetes-sigs/kubespray.git" # <-
    version: "v2.18.1"
```

**Comment by MusicDin** [comment]
This is now fixed in release [v2.0.7](https://github.com/MusicDin/kubitect/releases/tag/v2.0.7).
